docker/nginx-gateway.conf.template

```diff
@@ -71,6 +71,13 @@ server {
         internal;
     }
 
+    location = /logout {
+        add_header Clear-Site-Data "\"*\"";
+        add_header Cache-Control "no-store";
+        alias      /usr/share/nginx/html/online;
+        try_files  /redirectlogin.html /online/redirectlogin.html =504;
```

Comment:
Can't we just immediately redirect to login here?

Reply:
Oauth does not always use the `/auth/login` endpoint due to OS providing its own. So need to ensure that the endpoint that the app wants to send is communicated to the server from the client.